diff --unified --recursive --text mbedtls-3.3.0.ori/include/mbedtls/mbedtls_config.h mbedtls-3.3.0.new/include/mbedtls/mbedtls_config.h
--- mbedtls-3.3.0.ori/include/mbedtls/mbedtls_config.h	2022-12-14 20:06:39.000000000 +0100
+++ mbedtls-3.3.0.new/include/mbedtls/mbedtls_config.h	2022-12-15 10:44:51.062740700 +0100
@@ -1033,7 +1033,7 @@
  *
  * Uncomment this macro to disable the built-in platform entropy functions.
  */
-//#define MBEDTLS_NO_PLATFORM_ENTROPY
+#define MBEDTLS_NO_PLATFORM_ENTROPY
 
 /**
  * \def MBEDTLS_ENTROPY_FORCE_SHA256
